### PR TITLE
INF-48: Shift Displayr builds to Github Actions

### PR DESCRIPTION
**Pull request by mitchell-demol-displayr**
Changed the chrometests to compress with tgz and separated a large dll from it to speed up the process.
Have added code to allow both appveyor and github actions to run the tests accounting for theses changes.
Also changed the rerun download code to just be a single step.

### CODE REVIEW

File: .github/workflows/chrometest_run-batches.yml [hunk 1]
```diff
@@ -114,9 +119,28 @@ jobs:
           Set-ItemProperty -Path "HKCU:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Fonts" -Name "FontAwesome" -Value "$FontPath"
       - name: Download and Extract Release
         run: |
-          $uri = "${{ env.ENDPOINT_CHROMETEST }}/${{ env.BUILD_NUMBER }}/ChromeTests.7z?${{ secrets.ACCESS_TOKEN_CONTAINER_BUILDS }}"
-          Invoke-WebRequest -Uri $uri -OutFile ChromeTests.7z
-          7z x -y .\ChromeTests.7z
+          # retry in case of failed download
```
Comment (mitchell-demol-displayr) [inline .github/workflows/chrometest_run-batches.yml]: Rerun change was added after I made this branch, I just added this into a single step here

File: .github/workflows/chrometest_run-batches.yml [hunk 2]
```diff
@@ -142,6 +166,27 @@ jobs:
           .\nunit3-console.exe ChromeTests.dll --encoding=utf-8 --domain=None --inprocess --labels=All --testparam non-interactive=true `
             --result=${{ env.RERUN_RESULT_FILE }} `
             --testlist $OriginalFailTestResult
+      # Assuming average of 30 chrometests per day, total size of text files for a run should be ~300 kb, these will expire after 7 days, so 7x30x300 = 63 MB used at any time
```
Comment (mitchell-demol-displayr) [inline .github/workflows/chrometest_run-batches.yml]: Can remove this, basically just repeating the rerun step on job reruns. Need to cache the result file to do so.

File: .github/workflows/chrometest_run-batches.yml (hunk 1 above)
Comment (mlau-displayr) [reply .github/workflows/chrometest_run-batches.yml]: Nice

File: .github/workflows/chrometest_run-batches.yml (hunk 2 above)
Comment (mlau-displayr) [reply .github/workflows/chrometest_run-batches.yml]: We can do this a bit later.

Lets just make sure the chrometest are stable first. Cause we are making a big change atm

**Review by mlau-displayr — APPROVED** [review]

**Pull request closed without merging**